src/main/java/com/algorand/algosdk/v2/client/common/AlgodClient.java

```diff
@@ -17,6 +17,12 @@
 
 public class AlgodClient extends Client {
 
+    /**
+     * Construct a AlgodClient for communicating with the REST API.
+     * @param host host in a plain "localhost" format or a URI format like "https://localhost".
```

Comment:
I think "plain "localhost" format" is not a thing. It's essentially a URI without the protocol specified.  We should specify that we expect a URI and if the user doesn't contain the scheme (http/https), the client assumes HTTP.